### PR TITLE
Ipb 1121/withdraw referral

### PR DESCRIPTION
**Pull request by rgforsyth**
## What does this pull request do?

Adds the withdraw referral flow for pre-ICA referrals

## What is the intent behind these changes?

Allow referrals to be withdrawn

### CODE REVIEW

File: server/routes/appointments/feedback/shared/sessionFeedback/sessionFeedbackView.ts [hunk 1]
```diff
@@ -146,6 +146,7 @@ export default class SessionFeedbackView {
         },
         {
           divider: 'or',
+          class: 'govuk-label--m',
```
Comment (kth13) [inline server/routes/appointments/feedback/shared/sessionFeedback/sessionFeedbackView.ts:149]: Is this related to this change or you are fixing something that you found wrong?

File: server/utils/nunjucksSetup.ts [hunk 2]
```diff
@@ -24,4 +24,11 @@ export default function nunjucksSetup(app: express.Application, path: pathModule
   Object.keys(mojFilters).forEach(filterName => {
     njkEnv.addFilter(filterName, mojFilters[filterName])
   })
+
+  // eslint-disable-next-line func-names
+  njkEnv.addFilter('setAttribute', function (dictionary, key, value) {
```
Comment (kth13) [inline server/utils/nunjucksSetup.ts:29]: why we need add this filter

**Review by kth13 — COMMENTED** [review]
Overall looking good. Have got questions and some small suggestions

File: server/routes/appointments/feedback/shared/sessionFeedback/sessionFeedbackView.ts (hunk 1 above)
Comment (rgforsyth) [reply server/routes/appointments/feedback/shared/sessionFeedback/sessionFeedbackView.ts:149]: I think the default divider text was too small and didn't match the design.

File: server/utils/nunjucksSetup.ts (hunk 2 above)
Comment (rgforsyth) [reply server/utils/nunjucksSetup.ts:29]: This was to solve an issue of having multiple text boxes share the same data field. It allowed us to update them with unique id's and set the govuk text box for each radio within the njk file itself, whilst also keeping the radio list separated into different sections, but treated as one list.

**Pull request closed without merging**